app_check/integration_test/src/integration_test.cc

```diff
@@ -370,13 +370,18 @@ TEST_F(FirebaseAppCheckTest, TestGetTokenForcingRefresh) {
   EXPECT_NE(token.token, "");
   EXPECT_NE(token.expire_time_millis, 0);
 
+  // Wait a bit to make sure the expire time would be different
```

Comment:
What's the reason that this sleep is needed?

Reply:
Oh I see, if the two tokens are generated within a fraction of a second they may have identical expiration times. Seems reasonable.